core/blockchain.go

```diff
@@ -431,7 +434,10 @@ func (bc *BlockChain) unindexBlocks(tail uint64, head uint64, done chan struct{}
 // Meaning that this function should never be called.
 func (bc *BlockChain) dispatchTxUnindexer() {
 	defer bc.wg.Done()
-	txLookupLimit := bc.cacheConfig.TxLookupLimit
+	if bc.cacheConfig.TxLookupLimit < 0 {
```

Comment:
Because we check `TxLookupLimit` before spawning `dispatchTxUnindexer`, do we need to worry about this?

Reply:
responded: https://github.com/ava-labs/coreth/pull/400#discussion_r1400405969